woocommerce/woocommerce
# Refactor AttributeField into sub-components

### All Submissions:






### Changes proposed in this Pull Request:



This PR refactors `AttributeField` into sub-components, where those sub-components expose all UI strings as props, allowing them to be overridden as necessary. This will make creating an `OptionsField` (or, allowing `AttributeField` to switch between Attributes and Options) easier.

Part of #35770.






### How to test the changes in this Pull Request:



1. Go to Products > Add New (MVP)
2. Interact with the Attributes section and verify there are no regressions (there should be no change in functionality or UX with this PR).



### Other information:




### FOR PR REVIEWER ONLY:
